core/spec/spec_helper.rb

```diff
@@ -16,7 +16,7 @@
 require 'spree/testing_support/partial_double_verification'
 require 'spree/testing_support/silence_deprecations'
 require 'spree/testing_support/preferences'
-require 'spree/config'
+require 'spree'
```

Comment:
What's up with this change?

Reply:
Without it, `Spree::AppConfiguration` does not know about `Spree.deprecator`. I'm sure I can find another way of working around it :)